pycbc/workflow/pegasus_sites.py

```diff
@@ -223,21 +223,24 @@ def add_osg_site(sitecat, cp):
                       value="ON_SUCCESS")
     site.add_profiles(Namespace.CONDOR, key="success_exit_code",
                       value="0")
-    site.add_profiles(Namespace.CONDOR, key="+OpenScienceGrid",
+    site.add_profiles(Namespace.CONDOR, key="My.OpenScienceGrid",
                       value="True")
     site.add_profiles(Namespace.CONDOR, key="getenv",
                       value="False")
-    site.add_profiles(Namespace.CONDOR, key="+InitializeModulesEnv",
+    site.add_profiles(Namespace.CONDOR, key="ulog_execute_attrs",
+                      value="GLIDEIN_Site")
+    site.add_profiles(Namespace.CONDOR, key="My.InitializeModulesEnv",
                       value="False")
-    site.add_profiles(Namespace.CONDOR, key="+SingularityCleanEnv",
+    site.add_profiles(Namespace.CONDOR, key="My.SingularityCleanEnv",
                       value="True")
+    site.add_profiles(Namespace.CONDOR, key="My.DAGManNodesMask",
+                      value=r"\"0,1,2,4,5,7,8,9,10,11,12,13,16,17,24,27,35,36,40\"")
```

Comment:
I agree with Gareth about adding some extra documentation here, even just a link to where to find the meaning of these magic numbers, though I prefer comments in the code than notes in the PR.

Reply:
Yeah, the DAGManNodesMask doesn't mean a lot to me. I guess it just relates to filtering logging messages based on the docs though. 

https://htcondor.readthedocs.io/en/latest/classad-attributes/job-classad-attributes.html#DAGManNodesMask